packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -4196,8 +4162,7 @@ function commitPassiveUnmountEffectsInsideOfDeletedTree_begin(
     resetCurrentDebugFiberInDEV();
 
     const child = fiber.child;
-    // TODO: Only traverse subtree if it has a PassiveStatic flag. (But, if we
-    // do this, still need to handle `deletedTreeCleanUpLevel` correctly.)
```

Comment:
What does this mean? 

> But, if we do this, still need to handle `deletedTreeCleanUpLevel` correctly.

Reply:
Not quite sure, but that flag no longer exists, so I removed this part of the comment…